pkg/tidb/proxy.go

```diff
@@ -148,12 +149,12 @@ func (p *proxy) pickActiveConn() (out net.Conn) {
 // pick returns an active remote if there is any.
 func (p *proxy) pick() *remote {
 	var picked *remote
-	if p.current == "" {
```

Comment:
It is reported that it exists the data race between L#151 and L$156 when testing.